src/indexPage/react-components/binary-landing/Navigation.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { translate } from '../../../common/i18n';
+
+
+const MoveToDerivNav = () => (
+    <div className='navigation'>
+        <div className='navigation-container'>
+            <div class="navigation-logo">
+                <a href="#">
+                    <img src="image/binary.png" />
+                </a>
+            </div>
+            <a href="https://oauth.deriv.com/oauth2/authorize?app_id=16929&l=en&brand=deriv" className="navigation-to-deriv" rel="noopener noreferrer">
+                <button className='btn-group r-btn'>
```

Comment:
it was better and moe readable if we used complete word for variable names like `right-btn`